### ISSUE TITLE
Chrome 68: Network error synchronizing

### ISSUE THREAD

**Issue by dkaparis** [marcelklehr/floccus#138]
### Software versions

* Browser: Chromium 68.0.3440.75 (Official Build)
* Nextcloud: 13.0.4 (Snap)
* Nextcloud Bookmarks app: 0.11.0
* Floccus: 2.2.8, 3.0.0-beta5

### Steps to reproduce

1. Enter nc URL and credentials as prompted to create a new account
2. Click "Sync Now"

### Expected outcome

Bookmarks to be synchronized successfully

### Actual outcome

Error message: Network error: Check your network connection and your account details

Checking the network requests in the development console, I see a long series of requests to https://<URL_REDACTED>/index.php/apps/bookmarks/public/rest/v2/bookmark?page=-1 with 302 result and finally a failure: net::ERR_TOO_MANY_REDIRECTS

Here is a relevant part of the console output:
```
Nextcloud.js:411 GET https://<URL_REDACTED>/index.php/apps/bookmarks/public/rest/v2/bookmark?page=-1 0 ()
Account.js:168 Error: Network error: Check your network connection and your account details
    at NextcloudAdapter._callee14$ (Nextcloud.js:421)
    at tryCatch (polyfill.js:6900)
    at Generator.invoke [as _invoke] (polyfill.js:7138)
    at Generator.prototype.(anonymous function) [as throw] (chrome-extension://djejpebekaoommcjfeaiipdikmdjkblg/dist/js/polyfill.js:6952:21)
    at step (background-script.js:25144)
    at background-script.js:25144
Account.js:170 Syncing failed with Network error: Check your network connection and your account details
```

If I clear browser cookies, the sync succeeds once and on the next and subsequent tries the same error occurs.

**Comment by florealcab** [comment]
This problem occurs since chrome 68, for me too (for all I think). There is something broken with this new version.

Problem also present for passman (which also interacts with nextcloud...)

**Comment by marcelklehr** [comment]
mhh, any coherence among nextcloud versions?

**Comment by Larx** [comment]
I have the same problem. Chromium 68.0.3440.75 and Nexcloud 13.0.5, openSUSE packages for both.

**Comment by jlbprof** [comment]
My Nextcloud has a self-signed certificate and I get that from time to
time.   When that happens I just login to my Nextcloud Files section on
that browser and then I can sync.   I think Firefox allows you to remember
that certificate but Chrome/Chromium does not, so you have to redo the cert
periodically.

**Comment by Larx** [comment]
My certificate is a regular CertBot one, so it should be valid.

**Comment by marcelklehr** [comment]
for reference: The relevant passman issue is this one: https://github.com/nextcloud/passman-webextension/issues/269

**Comment by marcelklehr** [comment]
I can reproduce this with Version 68.0.3440.75 (Official Build)
However, it seems to be fixed with Version 68.0.3440.84 (Official Build) (64-bit)

**Comment by darrellittle** [comment]
I have the same issue on two separate PCs using Chrome Version 68.0.3440.84 (Official Build) (64-bit).
Firefox works fine on both PCs

**Comment by marcelklehr** [comment]
Mh, weird.

I've found the following issue which looks like it describes what we're experiencing. https://bugs.chromium.org/p/chromium/issues/detail?id=784528

Let's see, what'll happen.

**Comment by jlbprof** [comment]
I am getting the problem using beta6 and either a Nextcloud account or
Webdav, for a few days.   I think Nextcloud did something in a recent
update as I recently updated.

Latest Chrome (68.0.3440.84) on Mac
Also latest on Windows.

Nextcloud: 13.0.5 stable
It is on CentOS 7 updated this weekend.

I am gonna try a cPanel Account WebDAV account later today, will let you
know how that goes.

Oh I see the Chrome bug you just referenced that makes sense.  I will still
try a cPanel WebDAV later.

Julian

**Comment by jlbprof** [comment]
Is it possible there is a "new permission" that needs to be set in the
extension?

**Comment by marcelklehr** [comment]
@jlbprof I doubt it. At least the chrome changelog doesn't mention anything.

@florealcab I've tried adjusting the floccus code, but haven't had any luck so far, which makes me think it's a bug in chrome.

**Comment by marcelklehr** [comment]
I've just released v2.2.9 which should fix this. (It was a spec change of the fetch API, introduced in Chrome 68)

**Comment by jlbprof** [comment]
That worked for me, let me know when the beta is updated.   :)
